mat-process-api.Tests/V1/Controllers/ProcessDataImagesControllerTests.cs

```diff
@@ -175,5 +182,329 @@ public void given_an_exception_thrown_check_that_the_controller_throws_correct_e
             Assert.AreEqual(expectedStatusCode, result.StatusCode);
         }
         #endregion
+
+        #region Get Process Image
+
+        [Test]
+        public void given_a_valid_request_when_GetProcessImage_controller_method_is_called_then_it_returns_a_200_Ok_response()
+        {
+            //arrange
+            var request = MatProcessDataHelper.CreateGetProcessImageRequestObject();
+            var expectedStatusCode = 200;
+            _mockGetValidator.Setup(x => x.Validate(It.IsAny<GetProcessImageRequest>())).Returns(new FV.ValidationResult()); //setup validator to return a no error validation result
+
+            //act
+            var controllerResponse = _processImageController.GetProcessImage(request.processType,request.processRef,request.imageId,request.fileExtension);
```

Comment:
Related to comment '<1>' . For this test the contents of the request don't matter, I think this controller method call should be supplied with nulls to indicate that to avoid confusion about the test. Also it makes test harder to read and follow.

Like that:
`var controllerResponse = _processImageController.GetProcessImage(null, null, null ,null);`

Reply:
Obsolete due to model binding changes.